src/main/java/de/tum/in/www1/artemis/web/rest/dto/AtheneDTO.java

```diff
@@ -0,0 +1,37 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import java.util.*;
+
+import org.springframework.context.annotation.Profile;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.TextBlockType;
+import de.tum.in.www1.artemis.domain.TextCluster;
+
+@Profile("athene")
```

Comment:
Not sure what `@Profile` does here. But I am happy to keep it if it does anything.

Reply:
According to my research, it does not do anything in this case. I removed it